contracts/Voter.sol

```diff
@@ -239,7 +250,9 @@ contract Voter is IVoter {
         isAlive[_gauge] = true;
         _updateFor(_gauge);
         pools.push(_pool);
-        emit GaugeCreated(_gauge, msg.sender, _internal_bribe, _external_bribe, _pool);
+        Pair(_pool).setHasGauge(true); // may need to switch to IPair?
+        Pair(_pool).setExternalBribe(_wxbribe); // Changed this to wrapped external bribe  from
```

Comment:
```suggestion
        Pair(_pool).setExternalBribe(_wxbribe); // Changed this to wrapped external bribe
```

Reply:
btw yeah use `IPair`?

Reply:
yes.. that is likely a typo. must be `IPair`

Reply:
Raised in instruments' PR. Fixed.